pallets/gear/src/lib.rs

```diff
@@ -1903,6 +1961,72 @@ pub mod pallet {
 
             Ok(())
         }
+
```

Comment:
And as I mentioned before, you never paste here actual gas allowance (it also may be greater than block gas limit constant): should we calculate it in each extrinsic?

Reply:
Finally realized what you meant (having fixed failing tests). Yes, certainly - doing it this way now.